c_glib/arrow-glib/type.h

```diff
@@ -97,6 +99,7 @@ typedef enum {
   GARROW_TYPE_TIME64,
   GARROW_TYPE_MONTH_INTERVAL,
   GARROW_TYPE_DAY_TIME_INTERVAL,
+  GARROW_TYPE_DECIMAL64,
```

Comment:
Could you use the same order as `arrow::Type::type`? We want to use the same values as `arrow::Type::type`:
https://github.com/apache/arrow/blob/2df7b23af717f505c16fedbef6e4223ac163ac5a/c_glib/arrow-glib/type.h#L74

We can use ` = 44` for now:

https://github.com/apache/arrow/blob/2df7b23af717f505c16fedbef6e4223ac163ac5a/cpp/src/arrow/type_fwd.h#L464-L465

```diff
diff --git a/c_glib/arrow-glib/type.h b/c_glib/arrow-glib/type.h
index 6f33ad64ef..9c2843109e 100644
--- a/c_glib/arrow-glib/type.h
+++ b/c_glib/arrow-glib/type.h
@@ -113,6 +113,8 @@ typedef enum {
   GARROW_TYPE_LARGE_LIST,
   GARROW_TYPE_MONTH_DAY_NANO_INTERVAL,
   GARROW_TYPE_RUN_END_ENCODED,
+  /* TODO: Remove = 44 when we add STRING_VIEW..DECIMAL32. */
+  GARROW_TYPE_DECIMAL64 = 44,
 } GArrowType;
 
 /**
```

Reply:
Thanks Fixed. What does `STRING_VIEW` mean by the way?

Reply:
https://arrow.apache.org/docs/format/Columnar.html#variable-size-binary-view-layout

We should support it in GLib too.